docs/source/library-user-guide/functions/adding-udfs.md

```diff
@@ -419,6 +432,7 @@ impl ScalarUDFImpl for AsyncUpper {
         Ok(DataType::Utf8)
     }
 
+    // Note the normal invoke_with_args method is not called for Async UDFs
```

Comment:
i wish it didn't exist :(

Reply:
Yeah, we could potentially refactor the traits a bit more to avoid this, but I think that might cause more API churn than the value it added.

Reply:
We certainly should.
The functions we plug into LP shouldn't have to answer things like name, aliases, description, schema_name.
It's SQL frontend stuff, not the LP stuff.